lib/elixir/src/elixir_module.erl

```diff
@@ -351,7 +351,8 @@ attributes(DataSet, DataBag, PersistedAttributes) ->
 lookup_attribute(DataSet, DataBag, Key) when is_atom(Key) ->
   case ets:lookup(DataSet, Key) of
     [{Key, _, accumulate}] -> bag_lookup_element(DataBag, {accumulate, Key}, 2);
```

Comment:
We should either match `Key` in all of them or in none of them. None is fine, therefore:

```suggestion
    [{_, _, accumulate}] -> bag_lookup_element(DataBag, {accumulate, Key}, 2);
```